src/obfl-specification.html

```diff
@@ -1642,6 +1642,15 @@ <h4 id="L1105" class="include">toc-entry-on-resumed</h4>
           </ul></dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd><dl>
+        <dt>started-volume-first-content-page</dt>
+          <dd>The variable holding the first page number of the content of the
+            started volume. It can be used in an evaluate expression, e.g.
+            <code>&lt;evaluate
+            expression="$started-volume-first-content-page"&gt;</code>.</dd>
```

Comment:
Keeping the xml correct even if it's only in an example is preferable. Missing closing tag slash.

For an example
https://github.com/mtmse/obfl/blob/c64df35da3bfe12f4ebf71a25aed20799620ff28/src/obfl-specification.html#L1206

Reply:
Fixed. Thanks!